src/plots/plots.js

```diff
@@ -1335,7 +1339,11 @@ plots.supplyLayoutGlobalDefaults = function(layoutIn, layoutOut, formatObj) {
     coerce('title.y');
     coerce('title.yanchor');
 
-    coerce('title.subtitle.text', layoutOut._dfltTitle.subtitle);
+    var subtitleIn = layoutIn.title;
```

Comment:
@archmoj This must be the source of the bug -- should be `layoutIn.title.subtitle`